docs/lab/results/result-formats.mdx

```diff
@@ -101,6 +103,120 @@ Interpretation is a string value that can be one of the following:
 3. `Interpretation.CRITICAL` - The result is outside of critical parameters.
     In this case, refer to the [critical values](/lab/workflow/lab-test-lifecycle#critical-results) section.
 
+#### LOINC
+
+As it is possible to test the same biomarkers across different labs, there must be a way to match results across labs. For this, we follow the LOINC standard.
+
+In the `BiomarkerResult` object, you can see two fields `loinc_slug` and `loinc`. These fields refer to the LOINC standard, and are to be used for Customers to be able to match results across labs.
+
+You can expect that the `slug` field is what the lab returns to us, and the `loinc_slug` is the standardised version that will match across labs, if the marker in questions is the same.
+
+An example:
+
+| Lab | Slug | LOINC | LOINC Slug |
+| --- | --- | --- | --- |
+| Labcorp | hdl-cholesterol | 2085-9 | cholesterol-in-hdl-mass-vol |
+| USSL | hdl | 2085-9 | cholesterol-in-hdl-mass-vol |
+
+As you can see, the same biomarker `HDL Cholesterol` can have different slugs in different Labs, but be represented by the same LOINC.
```

Comment:
As you can see, the same biomarker `HDL Cholesterol` can have different slugs across different laboratories. However it's represented by the same LOINC value.